src/PollutantsComponent.js

```diff
@@ -50,8 +50,13 @@ function PollutantsComponent(props) {
             return data;
 
         for (let dataPoint of device.graph) {
+            let currPollutant = PollutantByID(pollutantId);
             if (typeof dataPoint[pollutantId] != 'undefined')
```

Comment:
I might be blind but is this if statement doing anything here? I don't see any enclosing `{}`

Reply:
Good catch, remnant of old code I think. Will adjust.